src/UID2.Client/IUID2Client.cs

```diff
@@ -34,10 +35,15 @@ public interface IUID2Client
         /// <returns>Response showing if decryption is successful and the resulting UID if successful.
         /// Or it could return error codes/string indicating what went wrong
         /// </returns>
+        [Obsolete("Please use Decrypt(string token) instead.")] 
         DecryptionResponse Decrypt(string token, DateTime now);
```

Comment:
should we rename the param to utcNow to make it obvious that it should be utc and state that in the param comment?

Reply:
can change the param but this is Obsolete, and Encrypt() in this class doesn't take a DateTime

Reply:
ah ok, i suggest to change the param name to make it clear for the 2 people out there but anyway nitpick